### PR TITLE
CBG-363: Add doc too large error

### PR DESCRIPTION
**Pull request by JRascagneres**
- [x] Merge https://github.com/couchbaselabs/walrus/pull/47
- [x] Bump manifest

## Integration Test
- [x] http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/1123/

### CODE REVIEW

**Comment by bbrks** [comment]
It would be great to copy `TestAddingAttachment` to generate a >20MB document (no attachment)

**Comment by JRascagneres** [comment]
> It would be great to copy `TestAddingAttachment` to generate a >20MB document (no attachment)

Very true... Will get that done!

File: manifest/default.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 
   <project name="sync_gateway_admin_ui" path="godeps/src/github.com/couchbaselabs/sync_gateway_admin_ui" revision="93c74bac9ddc2979ab895a37087c225c998b03bf" remote="couchbaselabs"/>
 
-  <project name="walrus" path="godeps/src/github.com/couchbaselabs/walrus" remote="couchbaselabs" revision="619dc4d3e720b63ebb3bac5c52f0407f66d358b7"/>
+  <project name="walrus" path="godeps/src/github.com/couchbaselabs/walrus" remote="couchbaselabs" revision="844a7348ba43bf7eb54eb8aa436990c71d949ec6"/>
```
Comment (bbrks) [inline manifest/default.xml]:
```suggestion
  <project name="walrus" path="godeps/src/github.com/couchbaselabs/walrus" remote="couchbaselabs" revision="62e1e281e9578eed39ca7f894cabae1faa07afc3"/>
```

**Review by bbrks — APPROVED** [review]

File: base/error.go [hunk 2]
```diff
@@ -114,6 +116,12 @@ func ErrorAsHTTPStatus(err error) (int, string) {
 			return http.StatusBadGateway, fmt.Sprintf("%s (%s)",
 				string(unwrappedErr.Body), unwrappedErr.Status.String())
 		}
+	case *gocbcore.KvError:
+		if unwrappedErr.Code == gocbcore.StatusTooBig {
```
Comment (adamcfraser) [inline base/error.go]: Can we try just checking for gocb.ErrTooBig inside the switch on unwrappedErr above?   I think that should be sufficient, and keeps all the gocb error checking in one place.

**Review by bbrks — APPROVED** [review]

**Review by bbrks — APPROVED** [review]

**Pull request merged**